diff --git a/source/thread_win32_osx.h b/source/thread_win32_osx.h
index d7f94ef5..b97dfd10 100644
--- a/source/thread_win32_osx.h
+++ b/source/thread_win32_osx.h
@@ -16,7 +16,7 @@
 // スレッドのデフォルトスタックサイズが小さい環境で、
 // 少なくとも8MB確保するためのthread生成のためのwrapper
 
-#if defined(__APPLE__) || defined(__MINGW32__) || defined(__MINGW64__) || defined(USE_PTHREADS)
+#if defined(__APPLE__) || defined(__MINGW32__) || defined(__MINGW64__) || defined(USE_PTHREADS) || defined(__ANDROID__)
 
     #include <pthread.h>
     #include <functional>
